mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp

```diff
@@ -213,8 +142,9 @@ void FSStoragePlugin_test::testDeleteAll()
     QCOMPARE( m_storage->m_objectHandlesMap.size(), m_storage->m_pathNamesMap.size() );
 
     delete m_storage;
+    m_storage = nullptr;
     system("rm -rf ~/.local/mtp");
-    system("rm -rf /tmp/mtptests");
+    QVERIFY( QDir(STORAGE1).removeRecursively() );
     initTestCase();
```

Comment:
Old, but calling initTestCase() from a test case (and then another test case) smells a bit bad.

Reply:
fixed